src/expressions/functions/string.ts

```diff
@@ -40,6 +41,18 @@ export function lTrim(original: Expr): CypherFunction {
     return new CypherFunction("lTrim", [original]);
 }
 
+/**
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/functions/string/)
```

Comment:
Potentially https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-normalize-with-normal-form is the more correct URL?

Reply:
I reckon https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-normalize would be the more accurate (as this one covers both), but then for consistency we should update all the links of each individual function here :sweat_smile:

Reply:
> I reckon https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-normalize would be the more accurate (as this one covers both), but then for consistency we should update all the links of each individual function here 😅

Oh I see, yeah maybe so in a different PR?

Reply:
Maybe, tbh we need to give a bit of thought on these links.

I added them because I think it is a great documentation tool to have these links in the editor, but we have no guarantee that these links will remain up (in fact, I'm sure some of these links are already broken)